### DISCUSSION TITLE
spec: Add progress option

### DISCUSSION THREAD

**Discussion by ghost** [FiloSottile/age#221] [Q&A] [answered]
Is it possible to add an option to show the progress?
If you implement it, you should not display it by default according to Unix philosophy.

**Comment by codesoap** [comment, marked as answer]
I think the most UNIX way to do this would be to utilise `pv`:
```
pv mydata | age -r age1df9vthfpykfsd4teaehwau7q8pt0f8d6sad22dgerqamnd43t32qzv54e5 -o mydata.age
```

**Reply by FiloSottile** [reply]
Yeah I recommend using `pv` for progress monitoring. It's the beauty of UNIX composable tools!
